test/modules/test_modules_isup.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+"""Tests for Sopel's ``remind`` plugin"""
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+import pytest
+
+from sopel.modules import isup
+
+
+VALID_SITE_URLS = (
+    # no TLD, no scheme
+    ('example', 'http://example.com'),
+    # no scheme
+    ('www.example.com', 'http://www.example.com'),
+    # with scheme
+    ('http://example.com', 'http://example.com'),
+    ('https://example.com', 'https://example.com'),
+    # wich scheme and URL path
+    ('http://example.com/path', 'http://example.com/path'),
+    ('https://example.com/path', 'https://example.com/path'),
+    # wich scheme, URL path, and query string
+    ('http://example.com/path?p=val', 'http://example.com/path?p=val'),
+    ('https://example.com/path?p=val', 'https://example.com/path?p=val'),
+    # not .com TLD
+    ('example.io', 'http://example.io'),
+    ('www.example.io', 'http://www.example.io'),
+)
+
+
+@pytest.mark.parametrize('site, expected', VALID_SITE_URLS)
+def test_get_site_url(site, expected):
+    assert isup.get_site_url(site) == expected
+
+
+INVVALID_SITE_URLS = (
```

Comment:
I didn't have as much time to review as expected, but I did have long enough to catch this.

```suggestion
INVALID_SITE_URLS = (
```

Reply:
Actual "lol wtf is this?" here. Good catch!

Reply:
Did a proper once-over tonight and this turned out to be the only thing worth noting. So, fix this silly typo, and we'll move forward. 😸